.github/workflows/test-sdks.yml

```diff
@@ -9,21 +9,10 @@ on:
 jobs:
   
   test-java-sdk:
-    runs-on: macos-latest
-    steps:
-      - name: Check out Java SDK
-        uses: actions/checkout@v3
-        with:
-          repository: 'Eppo-exp/java-server-sdk'
-      - name: Set up JDK 11
-        uses: actions/setup-java@v3
-        with:
-          java-version: '11'
-          distribution: 'adopt'
-      - name: 'Set up GCP SDK'
-        uses: 'google-github-actions/setup-gcloud@v0'
-      - name: Run tests
-        run: make test
+    uses: Eppo-exp/java-server-sdk/.github/workflows/lint-test-sdk.yml@tp/workflows/remote
```

Comment:
Is the eventual plan to remove this file and just use `test-sdks-remote`?

Reply:
the remote workflow is to trigger the tests to run in the respective SDK repos, effectively turning them red when the test data causes a test failure. The `test-sdks.yml` runs the SDK tests but in the sdk-test-data repo, so it doesn't cause spurious notifications from the repos.